google-cloud-examples/src/main/java/com/google/cloud/examples/bigtable/InstanceAdmin.java

```diff
@@ -0,0 +1,213 @@
+package com.google.cloud.examples.bigtable;
+
+import com.google.bigtable.admin.v2.ProjectName;
+import com.google.cloud.bigtable.admin.v2.BigtableInstanceAdminClient;
+import com.google.cloud.bigtable.admin.v2.BigtableInstanceAdminSettings;
+import com.google.cloud.bigtable.admin.v2.models.Cluster;
+import com.google.cloud.bigtable.admin.v2.models.CreateClusterRequest;
+import com.google.cloud.bigtable.admin.v2.models.CreateInstanceRequest;
+import com.google.cloud.bigtable.admin.v2.models.Instance;
+import com.google.cloud.bigtable.admin.v2.models.Instance.Type;
+import com.google.cloud.bigtable.admin.v2.models.StorageType;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
+public class InstanceAdmin {
+
+  public static void main(String[] args) throws IOException {
+
+    String GCLOUD_PROJECT = System.getenv("GCLOUD_PROJECT");
+
+    if (GCLOUD_PROJECT.length() == 0) {
+      throw new Error("Environment variables GCLOUD_PROJECT must be set!");
+    }
+
+    BigtableInstanceAdminSettings instanceAdminSettings =
+        BigtableInstanceAdminSettings.newBuilder().setProjectName(ProjectName.of(GCLOUD_PROJECT))
+            .build();
+
+    BigtableInstanceAdminClient adminClient =
+        BigtableInstanceAdminClient.create(instanceAdminSettings);
+
+    System.out.println("Create an instance (type: PRODUCTION) and run basic instance-operations");
+    runInstanceOperations(adminClient, "ssd-instance", "ssd-cluster");
+
+    System.out.println("Create DEVELOPMENT instance");
+    createDevInstance(adminClient, "hdd-instance", "hdd-cluster");
+
+    System.out.println("Delete the Instance");
+    deleteInstance(adminClient, "hdd-instance");
+
+    System.out.println("Add Cluster");
```

Comment:
The comments should be more semantic: not just adding a cluster, but enabling replication